src/lib/client.cc

```diff
@@ -238,7 +236,7 @@ Client::GetObject(const QString& bucket,
 		ds3Error = ds3_get_object(m_client, request,
 					  &caowi, write_to_file);
 	} else {
-		LOG_ERROR("GET object failed: unable to open file " + fileName);
+		LOG_ERROR("ERROR:       GET OBJECT failed, unable to open file "+fileName);
```

Comment:
Have the logging code handle logging "ERROR:" instead of forcing callers to always put it there.  For example, have Console::LogPrivate prepend the log message with "ERROR".  Same with "WARNING".